src/translator/helper.ts

```diff
@@ -35,6 +41,22 @@ export const createPrefillObject = (elements: TElementMap) => {
     return prefill;
 };
 
+export const translatePrefillObject = (
+    prefill: IPrefillConfig,
+    key: string
+): IPrefillAttributes[] => {
+    const prefillAttributes = prefill[key] as IPrefillAttributes[];
+    return prefillAttributes.filter((value) => {
```

Comment:
`prefillAttributes` may be undefined

```suggestion
    return prefillAttributes?.filter((value) => {
```